KsApi/MockService.swift

```diff
@@ -93,11 +93,13 @@
     fileprivate let fetchProjectResponse: Project?
     fileprivate let fetchProjectError: ErrorEnvelope?
 
-    fileprivate let fetchProjectNotificationsResponse: [ProjectNotification]
-
     fileprivate let fetchProjectsResponse: [Project]?
     fileprivate let fetchProjectsError: ErrorEnvelope?
 
+    fileprivate let fetchProjectCreatorDetailsResult: Result<ProjectCreatorDetailsEnvelope, GraphError>?
+
+    fileprivate let fetchProjectNotificationsResponse: [ProjectNotification]
```

Comment:
Alphabetized this so it moved around in a few places. But I now realize a bunch of things are still out of place so I might revert this change and rather fix the ordering separately.